automation/build.sbt

```diff
@@ -1,103 +1,12 @@
 import Settings._
 import sbt._
-import sbt.Keys.{fork, javaOptions, parallelExecution, testForkedParallel}
-import scala.collection.JavaConverters._
-
 
 lazy val leonardoTests = project.in(file("."))
   .settings(rootSettings:_*)
 
 version := "1.0"
 
-/**
```

Comment:
I removed forked JVMs because it adds complexity and we get better parallelism by using `ParallelTestExecution`.